src/config.h

```diff
@@ -147,6 +147,7 @@ class Config {
   Config(const std::string& filename);
 
   void updateFromTomlString(const std::string& contents);
+  static std::string readServerUrl(const std::string& provision_path);
```

Comment:
Would this code be more suited to being in the `Bootstrap` class?  To my mind `Config` is mostly about holding a set of configuration information for the system, and `Bootstrap` already reads the zip file to extract the x509 certificates.

Reply:
Yeah, probably. At this point, that function isn't dependent on anything in `Config` at all (hence why I made it `static`).